src/components/Bar/BarIndex.vue

```diff
@@ -139,6 +142,13 @@ export default {
         this.refreshBars()
     },
     methods: {
+        copyInviteLinkToClipboard(invite_code) {
+            if (Utils.copyToClipboard(`${window.location.origin}/bars/join/${invite_code}`)) {
+                this.$toast.success(this.$t('bars.invite-link-copied'))
```

Comment:
Currently, only `default` and `error` should be used.
```suggestion
                this.$toast.default(this.$t('bars.invite-link-copied'))
```